db/org_members.go

```diff
@@ -620,8 +620,10 @@ func (ms *MongoStorage) orgMembersByIDs(
 	if err != nil {
 		return 0, nil, err
 	}
-	totalPages := int(math.Ceil(float64(totalCount) / float64(pageSize)))
-
+	totalPages := 1 // by default, with pageSize == 0, we fetch all members in one page
+	if pageSize > 0 {
+		totalPages = int(math.Ceil(float64(totalCount) / float64(pageSize)))
```

Comment:
If `pageSize > totalCount` then we still get `totalPages=0` which also leads to weird API results, right?